scipy/sparse/linalg/eigen/tests/test_svds.py

```diff
@@ -16,7 +16,7 @@ def sorted_svd(m, k, which='LM'):
     # sorted.
     if isspmatrix(m):
         m = m.todense()
-    u, s, vh = svd(m)
+    u, s, vh = svd(m, full_matrices=False)
```

Comment:
By default `full_matrices` are `True` hence you are changing behavior here.

Reply:
This is one of the cases where having `full_matrices=True` by default bit the user. In this case, we are just using the reduced SVD, as if the user passed a `k` larger than the number of columns and rows, the slicing of `argsort` would throw.

Reply:
If you look at where it is used, it makes no difference since matrices are 3x4 and 4x4 in the test below. Note that these are test files and not the actual code. So it is used to verify svds behavior and I don't think it needs a fix.